formstack-baton-requests/cloud-formation.yaml

```diff
@@ -600,79 +600,33 @@ Resources:
         - |-
           {
             "Comment": "A step function for performing Formstack SARs",
```

Comment:
see screenshots in PR description to see the graphical representation of the changes to the step functions